src/main/scripts/addon

```diff
@@ -99,7 +99,7 @@ if [ -f "$PLF_HOME/addons/addons-manager.jar.new" ]; then
   mv "$PLF_HOME/addons/addons-manager.jar.new" "$PLF_HOME/addons/addons-manager.jar"
 fi
 
-JAVA_OPTS="-Dplf.home=\"$PLF_HOME\""
+JAVA_OPTS="$JAVA_OPTS -Dplf.home=\"$PLF_HOME\""
```

Comment:
I am not a big fan of reusing a so widely used environment variable because it could lead to many unpredictable problems in the case where the variable is already defined with some values.

I would really prefer we introduce a dedicated new EXO_AM_OPTS variable to get rid of this risk

@vsellier & @bourasom any advice on this ?

Reply:
Nothing more. I agree with the dedicated env variable usage